app/models/ZuoraCatalog/ZuoraCatalog.scala

```diff
@@ -41,6 +41,10 @@ case class ZuoraCatalogPricePlan(
 )
 
 object ZuoraCatalogPricePlan {
+
+  def round(price: Double, precision: Int): Double =
+    BigDecimal(price).setScale(2, BigDecimal.RoundingMode.HALF_DOWN).toDouble
```

Comment:
This was completely down to trial and error until the prices matched `subscriptions-frontend` and needs a proper investigation into how the prices are rounded